go/enclave/components/batch_registry.go

```diff
@@ -100,12 +100,12 @@ func (br *batchRegistry) OnBatchExecuted(batch *core.Batch, receipts types.Recei
 }
 
 func (br *batchRegistry) HasGenesisBatch() (bool, error) {
-	return br.headBatchSeq != nil, nil
+	return br.HeadBatchSeq() != nil, nil
```

Comment:
going through the function instead of accessing the variable directly.
It was useful to check whether that cached variable is correct